recipe/meta.yaml

```diff
@@ -3,12 +3,59 @@
 # Copyright (c) 2017, Continuum Analytics, Inc. All rights reserved.
 #
 # Distribution of the content approved by NVIDIA ( http://nvbugs/3052604 )
+{% set major_minor = "10.2" %}
 
-{% set version = "11.0.3" %}
-{% set version_patch = "450.51.06" %}  # [linux64]
-{% set version_patch = "450.51.06" %}  # [ppc64le]
-{% set version_patch = "451.82" %}  # [win]
-{% set major_minor = ".".join(version.split(".")[:2]) %}
+# The following cudavars dictionary is a table of metadata for selecting various
+# paramaters based on the X.Y version number of cudatoolkit. The following is a description
+# of the keys for each entry:
+#
+# version: the X.Y.Z version number for cudatoolkit.
+# version_patch: the patch/build number for the associated X.Y.Z number. This value
+#   is a dict of platform keys to patch numbers, because this value is platform dependent.
+# download_dir: The driectory in which a version's downloads exist. This does not
+#   follow a set pattern. The URL that is accessed for download is usually:
+#       https://developer.download.nvidia.com/compute/cuda/{{ download_dir }}
+# checksums: Another platform dependent key of the various download file checksums. For the
+#   runfiles, these are take from the official MD5 checksums. The official MD5 sums are
+#   usually available at the following 11.0+:
+#       https://developer.download.nvidia.com/compute/cuda/{{ download_dir }}/docs/sidebar/md5sum.txt
+
+{% set cudavars = {
+  "10.2": {
+    "version": "10.2.89",
+    "version_patch": {
+      "linux64": "440.33.01",
+      "ppc64le": "440.33.01",
+      "win": "441.22",
+    },
+    "download_dir": major_minor + "/Prod",
+    "checksums": {
+      "linux64": "3d816c583759efa8567a92c1a2af90db",
+      "ppc64le": "ecb6af432e64bc10e7a18cd6bd4be62f",
+      "win": "d9f5b9f24c3d3fc456a3c789f9b43419",
+    },
+  },
+  "11.0": {
```

Comment:
Oh sorry I missed this. So are we doing both of these in the same recipe then?

Reply:
No, just the metadata is there so that it is easy to switch the `major_minor` variable at the top and produce that version.

Reply:
This means that there shouldn't be a lot of conflicts when changing versions, if that makes sense.

Reply:
Ah ok. So having this in a different branch still makes sense. Is that right?

Reply:
Yeah I think it does